### PR TITLE
Added attributions for each image

### PR DESCRIPTION
**Pull request by pavpanchekha**

### CODE REVIEW

File: book/history.md [hunk 1]
```diff
@@ -20,12 +20,14 @@ other predecessors to HTML. (Except in this footnote!)
 The Memex concept
 =================
 
-<style>
-	figure { text-align: center; }
```
Comment (chrishtr) [inline book/history.md:24]: Why remove the centering?

File: book/history.md [hunk 2]
```diff
@@ -243,24 +248,28 @@ which built [Netscape
 Navigator](https://en.wikipedia.org/wiki/Netscape_Navigator), the first
 _commercial browser_,[^commercial-browser] which launched in 1994.
 
-<div style="display: grid; grid-template-columns: repeat(2, 1fr);">
+<div class="grid" style="display: grid; grid-template-columns: repeat(2, 1fr);">
```
Comment (chrishtr) [inline book/history.md:251]: What does this class name do?

File: book/history.md (hunk 1 above)
Comment (pavpanchekha) [reply book/history.md:24]: It is just moved to the general book stylesheet.

File: book/history.md (hunk 2 above)
Comment (pavpanchekha) [reply book/history.md:250]: Reduce margins in the contained images. Probably better to have a different HTML structure...

**Review by chrishtr — APPROVED** [review]

**Pull request merged**